src/Crowdin.Api/Reports/Match.cs

```diff
@@ -8,7 +8,7 @@ namespace Crowdin.Api.Reports
     public class Match
     {
         [JsonProperty("matchType")]
-        public MatchType MatchType { get; set; }
+        public MatchTypeObject MatchType { get; set; }
```

Comment:
This might be backwards incompatible, isn't it?

Reply:
Yes. We use a C# feature called implicit operator overloading to wrap original `MatchType` to unified `MatchTypeObject` and internally convert a constant to it's API value (in `[Description]`). As you can see in Reports unit tests (all of them continue working) - this change preserves backward compatibility